Source/VCGeneration/Split.cs

```diff
@@ -15,6 +15,14 @@ namespace VC
   using Bpl = Microsoft.Boogie;
   using System.Threading.Tasks;
 
+  public record SplitResult
```

Comment:
I don't love spreading the type "SplitResult" around, since it perpetuates the confusing overloading of "Split" as both the act of splitting larger VCs into smaller ones and the label used to refer to those smaller VCs themselves. Could we use "VerificationConditionResult" or "VCResult" instead? We could replace `int splitNum` with a more general "string vcIdentifier".

Reply:
I think that's a very good point. I'll rename both of those things.

Reply:
As for the identifier, I suspect it's going to make sense to add things to this record in the future, likely including a list of `ProofObligationDescription` from #505.

One thing to consider is that the XML file still uses the term `split`. We'll want to change that before long, too.